t3/utils/writer.py

```diff
@@ -204,13 +206,21 @@ def write_rmg_input_file(kwargs: dict,
     rmg_input += Template(model_template).render(**model)
 
     # simulator
-    simulator_template = """
-simulator(atol=${atol}, rtol=${rtol})
-"""
-    rmg_input += Template(simulator_template).render(atol=model_input['atol'], rtol=model_input['rtol'])
+    if t3 is not None and t3['sensitivity']:
```

Comment:
I think we should still check that the requested sensitivity adapter is RMG
(no harm to add these if it's not, but not usefull either)
What do you think?

Reply:
Sorry that was my mistake. I agree it shouldn't write unnecessary info. It now does the proper check for the RMG adapter with `if t3 is not None and t3['sensitivity']['adapter'] == 'RMG':`

Reply:
Ah my original fix was incorrect since I don’t think `t3` can ever be `None`. The `t3['options']` block is created by default even when running the `test_no_t3_no_qm` functional example which doesn’t pass any `t3` arguments. The new version instead checks if the `t3[‘sensitivity’]` block was defined and then if the `RMG` adapter was requested.